### PR TITLE
Add recipe for multiqc-plugins

### PR DESCRIPTION
**Pull request by fgypas**
Add recipe for multiqc-plugins

### CODE REVIEW

File: recipes/multiqc-plugins/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "multiqc-plugins" %}
```
Comment (druvus) [inline recipes/multiqc-plugins/meta.yaml:1]: Would it be possible to use a more specific (unique) name, such as "zavolan_multiqc_plugins"?

Comment (fgypas) [reply recipes/multiqc-plugins/meta.yaml]: Sure. I just renamed it.

**Comment by fgypas** [comment]
@druvus Can you please check if this is fine now?

**Comment by fgypas** [comment]
@BiocondaBot please review

**Comment by fgypas** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**